src/lib/common/sem.cpp

```diff
@@ -551,21 +550,14 @@ const char* timeStatSemGet(void)
 
 
 /* ****************************************************************************
```

Comment:
You have leaved the following in sem.cpp, but I think should be also moved:

* contexts_mutex_taken
* endpoint_mutexes_taken
* connectionContextSemGet()
* connectionSubContextSemGet()

and this file ends in line 549

Or maybe I'm missing something and they should be keep here?

Reply:
In fact in seems that contexts_mutex_taken and endpoint_mutexes_taken are already in the new file, so it is just a matter or removing it from src/lib/common/sem.cpp

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3869/commits/5f663bcfa195d8da60f4c74e939e7d8c19b4bb9d